### ISSUE TITLE
OptiFabric/Optifine is incopatibile with ReplayMod

### ISSUE THREAD

**Issue by music-cat-bread** [Chocohead/OptiFabric#537]
Hi.
I have encounter a problem with compatibility of OptiFabirc/Optifine
with ReplayMod, I am not sure where I should report this issue
first (I mean between here and replaymod).

Game Version: 1.18
Fabric Loader: 0.12.12
Fabric API: Newest for 1.18
Other mods: ModMenu
Optifine Version: 1.18_HD_U_H3

Log:
```
[17:58:49] [main/INFO]: Loading Minecraft 1.18 with Fabric Loader 0.12.12
[17:58:49] [main/INFO]: Fabric is preparing JARs on first launch, this may take a few seconds...
[17:58:59] [main/INFO]: Loading 51 mods:
	- fabric 0.43.1+1.18
	- fabric-api-base 0.4.1+b4f4f6cd14 via fabric
	- fabric-api-lookup-api-v1 1.3.5+3ac43d9514 via fabric
	- fabric-biome-api-v1 6.0.1+3ac43d9514 via fabric
	- fabric-blockrenderlayer-v1 1.1.9+3ac43d9514 via fabric
	- fabric-command-api-v1 1.1.6+3ac43d9514 via fabric
	- fabric-commands-v0 0.2.5+b4f4f6cd14 via fabric
	- fabric-containers-v0 0.1.18+d154e2c614 via fabric
	- fabric-content-registries-v0 0.4.5+6f53a73d14 via fabric
	- fabric-crash-report-info-v1 0.1.8+3ac43d9514 via fabric
	- fabric-dimensions-v1 2.1.7+43d2957114 via fabric
	- fabric-entity-events-v1 1.4.5+6b21378a14 via fabric
	- fabric-events-interaction-v0 0.4.15+3ac43d9514 via fabric
	- fabric-events-lifecycle-v0 0.2.5+b4f4f6cd14 via fabric
	- fabric-game-rule-api-v1 1.0.10+3ac43d9514 via fabric
	- fabric-item-api-v1 1.2.7+3ac43d9514 via fabric
	- fabric-item-groups-v0 0.3.3+3ac43d9514 via fabric
	- fabric-key-binding-api-v1 1.0.8+c8aba2f314 via fabric
	- fabric-keybindings-v0 0.2.6+b4f4f6cd14 via fabric
	- fabric-lifecycle-events-v1 1.4.10+c15ca33514 via fabric
	- fabric-loot-tables-v1 1.0.8+3ac43d9514 via fabric
	- fabric-mining-level-api-v1 1.0.3+3ac43d9514 via fabric
	- fabric-mining-levels-v0 0.1.7+b4f4f6cd14 via fabric
	- fabric-models-v0 0.3.3+3ac43d9514 via fabric
	- fabric-networking-api-v1 1.0.18+3ac43d9514 via fabric
	- fabric-networking-v0 0.3.5+b4f4f6cd14 via fabric
	- fabric-object-builder-api-v1 1.10.13+3ac43d9514 via fabric
	- fabric-object-builders-v0 0.7.7+3ac43d9514 via fabric
	- fabric-particles-v1 0.2.9+526dc1ac14 via fabric
	- fabric-registry-sync-v0 0.8.5+3ac43d9514 via fabric
	- fabric-renderer-api-v1 0.4.9+3ac43d9514 via fabric
	- fabric-renderer-indigo 0.4.12+3ac43d9514 via fabric
	- fabric-renderer-registries-v1 3.2.7+b4f4f6cd14 via fabric
	- fabric-rendering-data-attachment-v1 0.3.3+d154e2c614 via fabric
	- fabric-rendering-fluids-v1 0.1.18+3ac43d9514 via fabric
	- fabric-rendering-v0 1.1.9+b4f4f6cd14 via fabric
	- fabric-rendering-v1 1.10.3+6b21378a14 via fabric
	- fabric-resource-loader-v0 0.4.11+3ac43d9514 via fabric
	- fabric-screen-api-v1 1.0.7+3ac43d9514 via fabric
	- fabric-screen-handler-api-v1 1.1.11+3ac43d9514 via fabric
	- fabric-structure-api-v1 2.0.8+295197a714 via fabric
	- fabric-tag-extensions-v0 1.2.5+3ac43d9514 via fabric
	- fabric-textures-v0 1.0.9+3ac43d9514 via fabric
	- fabric-tool-attribute-api-v1 1.3.4+7de09f5514 via fabric
	- fabric-transfer-api-v1 1.5.4+b4f4f6cd14 via fabric
	- fabricloader 0.12.12
	- java 17
	- minecraft 1.18
	- mm 2.3 via optifabric
	- optifabric 1.12.7
	- replaymod 1.18.1-2.6.3
[17:58:59] [main/INFO]: SpongePowered MIXIN Subsystem Version=0.8.4 Source=file:/C:/Users/COVAND/AppData/Roaming/.minecraft/libraries/net/fabricmc/sponge-mixin/0.10.7+mixin.0.8.4/sponge-mixin-0.10.7+mixin.0.8.4.jar Service=Knot/Fabric Env=CLIENT
[17:59:00] [main/INFO]: Compatibility level set to JAVA_16
[17:59:00] [main/WARN]: Reference map 'mixins.nonmmlauncher.replaymod.refmap.json' for mixins.nonmmlauncher.replaymod.json could not be read. If this is a development environment you can ignore this message
[17:59:01] [main/WARN]: Error loading class: shadersmod/client/ShadersRender (java.lang.ClassNotFoundException: shadersmod/client/ShadersRender)
[17:59:01] [main/WARN]: Error loading class: net/coderbot/iris/uniforms/CommonUniforms (java.lang.ClassNotFoundException: net/coderbot/iris/uniforms/CommonUniforms)
[17:59:01] [main/WARN]: Error loading class: net/coderbot/iris/Iris (java.lang.ClassNotFoundException: net/coderbot/iris/Iris)
[17:59:01] [main/WARN]: Error loading class: shadersmod/client/ShadersRender (java.lang.ClassNotFoundException: shadersmod/client/ShadersRender)
[17:59:01] [main/WARN]: Error loading class: shadersmod/client/ShadersRender (java.lang.ClassNotFoundException: shadersmod/client/ShadersRender)
[17:59:04] [main/ERROR]: Mixin apply for mod replaymod failed mixins.jgui.json:MixinKeyboardListener -> net.minecraft.class_309: org.spongepowered.asm.mixin.injection.throwables.InvalidInjectionException Invalid descriptor on mixins.jgui.json:MixinKeyboardListener->@Inject::keyPressed(ILnet/minecraft/class_437;[ZIIILorg/spongepowered/asm/mixin/injection/callback/CallbackInfo;)V from mod replaymod! Expected (I[ZIIILnet/minecraft/class_437;Lorg/spongepowered/asm/mixin/injection/callback/CallbackInfo;)V but found (ILnet/minecraft/class_437;[ZIIILorg/spongepowered/asm/mixin/injection/callback/CallbackInfo;)V [INJECT Applicator Phase -> mixins.jgui.json:MixinKeyboardListener -> Apply Injections ->  -> Inject -> mixins.jgui.json:MixinKeyboardListener->@Inject::keyPressed(ILnet/minecraft/class_437;[ZIIILorg/spongepowered/asm/mixin/injection/callback/CallbackInfo;)V from mod replaymod]
org.spongepowered.asm.mixin.injection.throwables.InvalidInjectionException: Invalid descriptor on mixins.jgui.json:MixinKeyboardListener->@Inject::keyPressed(ILnet/minecraft/class_437;[ZIIILorg/spongepowered/asm/mixin/injection/callback/CallbackInfo;)V from mod replaymod! Expected (I[ZIIILnet/minecraft/class_437;Lorg/spongepowered/asm/mixin/injection/callback/CallbackInfo;)V but found (ILnet/minecraft/class_437;[ZIIILorg/spongepowered/asm/mixin/injection/callback/CallbackInfo;)V [INJECT Applicator Phase -> mixins.jgui.json:MixinKeyboardListener -> Apply Injections ->  -> Inject -> mixins.jgui.json:MixinKeyboardListener->@Inject::keyPressed(ILnet/minecraft/class_437;[ZIIILorg/spongepowered/asm/mixin/injection/callback/CallbackInfo;)V from mod replaymod]
	at org.spongepowered.asm.mixin.injection.callback.CallbackInjector.inject(CallbackInjector.java:565) ~[sponge-mixin-0.10.7+mixin.0.8.4.jar:0.10.7+mixin.0.8.4]
	at org.spongepowered.asm.mixin.injection.callback.CallbackInjector.inject(CallbackInjector.java:495) ~[sponge-mixin-0.10.7+mixin.0.8.4.jar:0.10.7+mixin.0.8.4]
	at org.spongepowered.asm.mixin.injection.code.Injector.inject(Injector.java:276) ~[sponge-mixin-0.10.7+mixin.0.8.4.jar:0.10.7+mixin.0.8.4]
	at org.spongepowered.asm.mixin.injection.struct.InjectionInfo.inject(InjectionInfo.java:445) ~[sponge-mixin-0.10.7+mixin.0.8.4.jar:0.10.7+mixin.0.8.4]
	at org.spongepowered.asm.mixin.transformer.MixinTargetContext.applyInjections(MixinTargetContext.java:1374) ~[sponge-mixin-0.10.7+mixin.0.8.4.jar:0.10.7+mixin.0.8.4]
	at org.spongepowered.asm.mixin.transformer.MixinApplicatorStandard.applyInjections(MixinApplicatorStandard.java:1052) ~[sponge-mixin-0.10.7+mixin.0.8.4.jar:0.10.7+mixin.0.8.4]
	at org.spongepowered.asm.mixin.transformer.MixinApplicatorStandard.applyMixin(MixinApplicatorStandard.java:400) ~[sponge-mixin-0.10.7+mixin.0.8.4.jar:0.10.7+mixin.0.8.4]
	at org.spongepowered.asm.mixin.transformer.MixinApplicatorStandard.apply(MixinApplicatorStandard.java:325) ~[sponge-mixin-0.10.7+mixin.0.8.4.jar:0.10.7+mixin.0.8.4]
	at org.spongepowered.asm.mixin.transformer.TargetClassContext.apply(TargetClassContext.java:421) ~[sponge-mixin-0.10.7+mixin.0.8.4.jar:0.10.7+mixin.0.8.4]
	at org.spongepowered.asm.mixin.transformer.TargetClassContext.applyMixins(TargetClassContext.java:403) ~[sponge-mixin-0.10.7+mixin.0.8.4.jar:0.10.7+mixin.0.8.4]
	at org.spongepowered.asm.mixin.transformer.MixinProcessor.applyMixins(MixinProcessor.java:363) [sponge-mixin-0.10.7+mixin.0.8.4.jar:0.10.7+mixin.0.8.4]
	at org.spongepowered.asm.mixin.transformer.MixinTransformer.transformClass(MixinTransformer.java:234) [sponge-mixin-0.10.7+mixin.0.8.4.jar:0.10.7+mixin.0.8.4]
	at org.spongepowered.asm.mixin.transformer.MixinTransformer.transformClassBytes(MixinTransformer.java:202) [sponge-mixin-0.10.7+mixin.0.8.4.jar:0.10.7+mixin.0.8.4]
	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.getPostMixinClassByteArray(KnotClassDelegate.java:247) [fabric-loader-0.12.12.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.tryLoadClass(KnotClassDelegate.java:150) [fabric-loader-0.12.12.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClassLoader.loadClass(KnotClassLoader.java:155) [fabric-loader-0.12.12.jar:?]
	at java.lang.ClassLoader.loadClass(ClassLoader.java:520) [?:?]
	at java.lang.Class.getDeclaredFields0(Native Method) ~[?:?]
	at java.lang.Class.privateGetDeclaredFields(Class.java:3297) [?:?]
	at java.lang.Class.getDeclaredFields(Class.java:2371) [?:?]
	at net.optifine.reflect.FieldLocatorTypes.<init>(FieldLocatorTypes.java:25) [Optifine-mapped.jar:?]
	at net.optifine.reflect.Reflector.<clinit>(Reflector.java:502) [Optifine-mapped.jar:?]
	at net.minecraft.class_128.method_568(class_128.java:173) [client-intermediary.jar:?]
	at net.minecraft.class_128.method_24305(class_128.java:343) [client-intermediary.jar:?]
	at net.minecraft.client.main.Main.main(Main.java:151) [client-intermediary.jar:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77) ~[?:?]
	at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
	at java.lang.reflect.Method.invoke(Method.java:568) ~[?:?]
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:608) [fabric-loader-0.12.12.jar:?]
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:77) [fabric-loader-0.12.12.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23) [fabric-loader-0.12.12.jar:?]
[17:59:04] [main/ERROR]: Minecraft has crashed!
net.fabricmc.loader.impl.FormattedException: java.lang.ExceptionInInitializerError
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:610) ~[fabric-loader-0.12.12.jar:?]
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:77) [fabric-loader-0.12.12.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23) [fabric-loader-0.12.12.jar:?]
Caused by: java.lang.ExceptionInInitializerError
	at net.minecraft.class_128.method_568(class_128.java:173) ~[client-intermediary.jar:?]
	at net.minecraft.class_128.method_24305(class_128.java:343) ~[client-intermediary.jar:?]
	at net.minecraft.client.main.Main.main(Main.java:151) ~[fabric-loader-0.12.12-1.18.jar:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77) ~[?:?]
	at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
	at java.lang.reflect.Method.invoke(Method.java:568) ~[?:?]
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:608) ~[fabric-loader-0.12.12.jar:?]
	... 2 more
Caused by: java.lang.RuntimeException: Mixin transformation of net.minecraft.class_309 failed
	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.getPostMixinClassByteArray(KnotClassDelegate.java:252) ~[fabric-loader-0.12.12.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.tryLoadClass(KnotClassDelegate.java:150) ~[fabric-loader-0.12.12.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClassLoader.loadClass(KnotClassLoader.java:155) ~[fabric-loader-0.12.12.jar:?]
	at java.lang.ClassLoader.loadClass(ClassLoader.java:520) ~[?:?]
	at java.lang.Class.getDeclaredFields0(Native Method) ~[?:?]
	at java.lang.Class.privateGetDeclaredFields(Class.java:3297) ~[?:?]
	at java.lang.Class.getDeclaredFields(Class.java:2371) ~[?:?]
	at net.optifine.reflect.FieldLocatorTypes.<init>(FieldLocatorTypes.java:25) ~[Optifine-mapped.jar:?]
	at net.optifine.reflect.Reflector.<clinit>(Reflector.java:502) ~[Optifine-mapped.jar:?]
	at net.minecraft.class_128.method_568(class_128.java:173) ~[client-intermediary.jar:?]
	at net.minecraft.class_128.method_24305(class_128.java:343) ~[client-intermediary.jar:?]
	at net.minecraft.client.main.Main.main(Main.java:151) ~[fabric-loader-0.12.12-1.18.jar:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77) ~[?:?]
	at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
	at java.lang.reflect.Method.invoke(Method.java:568) ~[?:?]
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:608) ~[fabric-loader-0.12.12.jar:?]
	... 2 more
Caused by: org.spongepowered.asm.mixin.transformer.throwables.MixinTransformerError: An unexpected critical error was encountered
	at org.spongepowered.asm.mixin.transformer.MixinProcessor.applyMixins(MixinProcessor.java:392) ~[sponge-mixin-0.10.7+mixin.0.8.4.jar:0.10.7+mixin.0.8.4]
	at org.spongepowered.asm.mixin.transformer.MixinTransformer.transformClass(MixinTransformer.java:234) ~[sponge-mixin-0.10.7+mixin.0.8.4.jar:0.10.7+mixin.0.8.4]
	at org.spongepowered.asm.mixin.transformer.MixinTransformer.transformClassBytes(MixinTransformer.java:202) ~[sponge-mixin-0.10.7+mixin.0.8.4.jar:0.10.7+mixin.0.8.4]
	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.getPostMixinClassByteArray(KnotClassDelegate.java:247) ~[fabric-loader-0.12.12.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.tryLoadClass(KnotClassDelegate.java:150) ~[fabric-loader-0.12.12.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClassLoader.loadClass(KnotClassLoader.java:155) ~[fabric-loader-0.12.12.jar:?]
	at java.lang.ClassLoader.loadClass(ClassLoader.java:520) ~[?:?]
	at java.lang.Class.getDeclaredFields0(Native Method) ~[?:?]
	at java.lang.Class.privateGetDeclaredFields(Class.java:3297) ~[?:?]
	at java.lang.Class.getDeclaredFields(Class.java:2371) ~[?:?]
	at net.optifine.reflect.FieldLocatorTypes.<init>(FieldLocatorTypes.java:25) ~[Optifine-mapped.jar:?]
	at net.optifine.reflect.Reflector.<clinit>(Reflector.java:502) ~[Optifine-mapped.jar:?]
	at net.minecraft.class_128.method_568(class_128.java:173) ~[client-intermediary.jar:?]
	at net.minecraft.class_128.method_24305(class_128.java:343) ~[client-intermediary.jar:?]
	at net.minecraft.client.main.Main.main(Main.java:151) ~[fabric-loader-0.12.12-1.18.jar:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77) ~[?:?]
	at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
	at java.lang.reflect.Method.invoke(Method.java:568) ~[?:?]
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:608) ~[fabric-loader-0.12.12.jar:?]
	... 2 more
Caused by: org.spongepowered.asm.mixin.throwables.MixinApplyError: Mixin [mixins.jgui.json:MixinKeyboardListener] from phase [DEFAULT] in config [mixins.jgui.json] from mod [replaymod] FAILED during APPLY
	at org.spongepowered.asm.mixin.transformer.MixinProcessor.handleMixinError(MixinProcessor.java:638) ~[sponge-mixin-0.10.7+mixin.0.8.4.jar:0.10.7+mixin.0.8.4]
	at org.spongepowered.asm.mixin.transformer.MixinProcessor.handleMixinApplyError(MixinProcessor.java:589) ~[sponge-mixin-0.10.7+mixin.0.8.4.jar:0.10.7+mixin.0.8.4]
	at org.spongepowered.asm.mixin.transformer.MixinProcessor.applyMixins(MixinProcessor.java:379) ~[sponge-mixin-0.10.7+mixin.0.8.4.jar:0.10.7+mixin.0.8.4]
	at org.spongepowered.asm.mixin.transformer.MixinTransformer.transformClass(MixinTransformer.java:234) ~[sponge-mixin-0.10.7+mixin.0.8.4.jar:0.10.7+mixin.0.8.4]
	at org.spongepowered.asm.mixin.transformer.MixinTransformer.transformClassBytes(MixinTransformer.java:202) ~[sponge-mixin-0.10.7+mixin.0.8.4.jar:0.10.7+mixin.0.8.4]
	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.getPostMixinClassByteArray(KnotClassDelegate.java:247) ~[fabric-loader-0.12.12.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.tryLoadClass(KnotClassDelegate.java:150) ~[fabric-loader-0.12.12.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClassLoader.loadClass(KnotClassLoader.java:155) ~[fabric-loader-0.12.12.jar:?]
	at java.lang.ClassLoader.loadClass(ClassLoader.java:520) ~[?:?]
	at java.lang.Class.getDeclaredFields0(Native Method) ~[?:?]
	at java.lang.Class.privateGetDeclaredFields(Class.java:3297) ~[?:?]
	at java.lang.Class.getDeclaredFields(Class.java:2371) ~[?:?]
	at net.optifine.reflect.FieldLocatorTypes.<init>(FieldLocatorTypes.java:25) ~[Optifine-mapped.jar:?]
	at net.optifine.reflect.Reflector.<clinit>(Reflector.java:502) ~[Optifine-mapped.jar:?]
	at net.minecraft.class_128.method_568(class_128.java:173) ~[client-intermediary.jar:?]
	at net.minecraft.class_128.method_24305(class_128.java:343) ~[client-intermediary.jar:?]
	at net.minecraft.client.main.Main.main(Main.java:151) ~[fabric-loader-0.12.12-1.18.jar:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77) ~[?:?]
	at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
	at java.lang.reflect.Method.invoke(Method.java:568) ~[?:?]
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:608) ~[fabric-loader-0.12.12.jar:?]
	... 2 more
Caused by: org.spongepowered.asm.mixin.injection.throwables.InvalidInjectionException: Invalid descriptor on mixins.jgui.json:MixinKeyboardListener->@Inject::keyPressed(ILnet/minecraft/class_437;[ZIIILorg/spongepowered/asm/mixin/injection/callback/CallbackInfo;)V from mod replaymod! Expected (I[ZIIILnet/minecraft/class_437;Lorg/spongepowered/asm/mixin/injection/callback/CallbackInfo;)V but found (ILnet/minecraft/class_437;[ZIIILorg/spongepowered/asm/mixin/injection/callback/CallbackInfo;)V [INJECT Applicator Phase -> mixins.jgui.json:MixinKeyboardListener -> Apply Injections ->  -> Inject -> mixins.jgui.json:MixinKeyboardListener->@Inject::keyPressed(ILnet/minecraft/class_437;[ZIIILorg/spongepowered/asm/mixin/injection/callback/CallbackInfo;)V from mod replaymod]
	at org.spongepowered.asm.mixin.injection.callback.CallbackInjector.inject(CallbackInjector.java:565) ~[sponge-mixin-0.10.7+mixin.0.8.4.jar:0.10.7+mixin.0.8.4]
	at org.spongepowered.asm.mixin.injection.callback.CallbackInjector.inject(CallbackInjector.java:495) ~[sponge-mixin-0.10.7+mixin.0.8.4.jar:0.10.7+mixin.0.8.4]
	at org.spongepowered.asm.mixin.injection.code.Injector.inject(Injector.java:276) ~[sponge-mixin-0.10.7+mixin.0.8.4.jar:0.10.7+mixin.0.8.4]
	at org.spongepowered.asm.mixin.injection.struct.InjectionInfo.inject(InjectionInfo.java:445) ~[sponge-mixin-0.10.7+mixin.0.8.4.jar:0.10.7+mixin.0.8.4]
	at org.spongepowered.asm.mixin.transformer.MixinTargetContext.applyInjections(MixinTargetContext.java:1374) ~[sponge-mixin-0.10.7+mixin.0.8.4.jar:0.10.7+mixin.0.8.4]
	at org.spongepowered.asm.mixin.transformer.MixinApplicatorStandard.applyInjections(MixinApplicatorStandard.java:1052) ~[sponge-mixin-0.10.7+mixin.0.8.4.jar:0.10.7+mixin.0.8.4]
	at org.spongepowered.asm.mixin.transformer.MixinApplicatorStandard.applyMixin(MixinApplicatorStandard.java:400) ~[sponge-mixin-0.10.7+mixin.0.8.4.jar:0.10.7+mixin.0.8.4]
	at org.spongepowered.asm.mixin.transformer.MixinApplicatorStandard.apply(MixinApplicatorStandard.java:325) ~[sponge-mixin-0.10.7+mixin.0.8.4.jar:0.10.7+mixin.0.8.4]
	at org.spongepowered.asm.mixin.transformer.TargetClassContext.apply(TargetClassContext.java:421) ~[sponge-mixin-0.10.7+mixin.0.8.4.jar:0.10.7+mixin.0.8.4]
	at org.spongepowered.asm.mixin.transformer.TargetClassContext.applyMixins(TargetClassContext.java:403) ~[sponge-mixin-0.10.7+mixin.0.8.4.jar:0.10.7+mixin.0.8.4]
	at org.spongepowered.asm.mixin.transformer.MixinProcessor.applyMixins(MixinProcessor.java:363) ~[sponge-mixin-0.10.7+mixin.0.8.4.jar:0.10.7+mixin.0.8.4]
	at org.spongepowered.asm.mixin.transformer.MixinTransformer.transformClass(MixinTransformer.java:234) ~[sponge-mixin-0.10.7+mixin.0.8.4.jar:0.10.7+mixin.0.8.4]
	at org.spongepowered.asm.mixin.transformer.MixinTransformer.transformClassBytes(MixinTransformer.java:202) ~[sponge-mixin-0.10.7+mixin.0.8.4.jar:0.10.7+mixin.0.8.4]
	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.getPostMixinClassByteArray(KnotClassDelegate.java:247) ~[fabric-loader-0.12.12.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.tryLoadClass(KnotClassDelegate.java:150) ~[fabric-loader-0.12.12.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClassLoader.loadClass(KnotClassLoader.java:155) ~[fabric-loader-0.12.12.jar:?]
	at java.lang.ClassLoader.loadClass(ClassLoader.java:520) ~[?:?]
	at java.lang.Class.getDeclaredFields0(Native Method) ~[?:?]
	at java.lang.Class.privateGetDeclaredFields(Class.java:3297) ~[?:?]
	at java.lang.Class.getDeclaredFields(Class.java:2371) ~[?:?]
	at net.optifine.reflect.FieldLocatorTypes.<init>(FieldLocatorTypes.java:25) ~[Optifine-mapped.jar:?]
	at net.optifine.reflect.Reflector.<clinit>(Reflector.java:502) ~[Optifine-mapped.jar:?]
	at net.minecraft.class_128.method_568(class_128.java:173) ~[client-intermediary.jar:?]
	at net.minecraft.class_128.method_24305(class_128.java:343) ~[client-intermediary.jar:?]
	at net.minecraft.client.main.Main.main(Main.java:151) ~[fabric-loader-0.12.12-1.18.jar:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77) ~[?:?]
	at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
	at java.lang.reflect.Method.invoke(Method.java:568) ~[?:?]
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:608) ~[fabric-loader-0.12.12.jar:?]
	... 2 more

```

**Comment by music-cat-bread** [comment]
I want note that then I removed Replay Mod and tried to join to a multiplayer server it crashed again

**Comment by FileOfName** [comment]
fixed in optifabric 1.12.10

**Comment by Kenocrystal** [comment]
> fixed in optifabric 1.12.10

its not, I just checked
